TodoApi/Middlewares/GlobalExceptionHandler.cs

```diff
@@ -19,22 +21,43 @@ public async Task InvokeAsync(HttpContext httpContext)
         {
             await _next(httpContext);
         }
+        catch(BadHttpRequestException ex)
+        {
+            await BadRequestHandlerException(httpContext, ex);
+        }
         catch (Exception ex)
         {
-            await HandleExceptionAsync(httpContext, ex);
+            await GlobalHandlerException(httpContext, ex);
         }
     }
 
-    private Task HandleExceptionAsync(HttpContext context, Exception exception)
+    private Task BadRequestHandlerException(HttpContext context, BadHttpRequestException exception)
+    {
+        _logger.LogError(exception.Message);
+        context.Response.ContentType = "application/json";
+        context.Response.StatusCode = (int)HttpStatusCode.InternalServerError;
+
+        var response = new
+        {
+            StatusCode = context.Response.StatusCode, //maybe its useless
```

Comment:
It's useless when you just copy code from HttpContext, better to use 400 (Bad Request) code here